app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -169,6 +170,14 @@ class DefaultToolbarMenu(
         if (shouldReverseItems) { menuItems.reversed() } else { menuItems }
     }
 
+    private val addons = BrowserMenuImageText(
+        context.getString(R.string.browser_menu_addon_manager), // need localization
+        R.drawable.ic_settings, // need icon
```

Comment:
We could use icon `ui-icons/mozac_ic_extensions.xml` like [add_ons_item.xml](https://github.com/mozilla-mobile/android-components/blob/bda375105f54479274269b2a4b1577feb624c26e/samples/browser/src/main/res/layout/add_ons_item.xml#L30) is using it.

Reply:
We can take care of it in follow up